python/lib/dcos/tests/test_util.py

```diff
@@ -16,6 +16,14 @@ def test_open_file():
         in str(excinfo.value)
 
 
+def test_read_file_secure_with_trailing_newline():
```

Comment:
Please add a test with multiple newlines at the end. I imagine that this is also easily accidentally done.

Reply:
Definitely a good test case. Should pass without modifications when using `rstrip` https://docs.python.org/3/library/stdtypes.html?highlight=rstrip#str.rstrip